docs/source/reference/kubernetes/kubernetes-getting-started.rst

```diff
@@ -119,6 +119,57 @@ Once your cluster administrator has :ref:`setup a Kubernetes cluster <kubernetes
     $ kubectl config set-context --current --namespace=mynamespace
 
 
+
+Viewing cluster status
```

Comment:
Q: thinking behind using 2 titles for the same ~ similar content on 2 pages? Shall we keep this one?